spec/features/manage_cmr/provider_identity_permissions/adding_provider_permissions_from_group_spec.rb

```diff
@@ -0,0 +1,232 @@
+describe 'Saving Provider Object Permissions from the group show page', reset_provider: true do
+  before :all do
+    @provider_group_name = 'Test Group for Creating Provider Object Permissions from group page'
+    @provider_group_description = 'Group for creating provider object permissions'
+    @provider_group = create_group(
+      name: @provider_group_name,
+      description: @provider_group_description,
+      provider_id: 'MMT_2'
+    )
+
+    wait_for_cmr
+
+    @system_group_name = 'Test System Group for Provider Permissions from group page'
+    @system_group_description = 'System Group for creating provider permissions'
+    @system_group = create_group(
+      name: @system_group_name,
+      description: @system_group_description,
+      provider_id: nil,
+      admin: true
+    )
+
+    wait_for_cmr
+  end
+
+  after :all do
```

Comment:
If this block fails down the line, for whatever reason, I think it will do so silently and the persistent permissions may cause other tests to fail. I think the responses should be verified.

Reply:
modified to use helpers, added error throwing to some helper methods